test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/Profiler/ProfilerServiceTests.cs

```diff
@@ -14,72 +14,154 @@
+                    var stopParams = new StopProfilingParams()
+                    {
+                        OwnerUri = connectionResult.ConnectionInfo.OwnerUri
+                    };
+
+                    var stopContext = new Mock<RequestContext<StopProfilingResult>>();
+                    stopContext.Setup(rc => rc.SendResult(It.IsAny<StopProfilingResult>()))
+                        .Returns(Task.FromResult(0));
+
+                    await profilerService.HandleStopProfilingRequest(stopParams, stopContext.Object);
+
+                    xeSession.Refresh();
+                    Assert.That(xeSession.IsRunning, Is.False, "Session should be stopped due to HandleStopProfilingRequest");
+                    stopContext.VerifyAll();
+                }
+                finally
+                {
+                    try
+                    {
+                        xeStore.Sessions.Refresh();
+                        if (xeStore.Sessions.Contains(sessionName))
+                        {
+                            try
+                            {
+                                xeStore.Sessions[sessionName].Stop();
+                            }
+                            catch 
+                            { }
+                            xeStore.Sessions[sessionName].Drop();
+                        }
+                    }
+                    catch
+                    { }
+                }
             }
-        }
-
-        /// <summary>
-        /// Verify  the profiler service XEvent session factory
-        /// </summary>
-        //[Test]
-        public void TestCreateXEventSession()
-        {
-            var liveConnection = LiveConnectionHelper.InitLiveConnectionInfo("master");
-            ProfilerService profilerService = new ProfilerService();
-            IXEventSession xeSession = profilerService.GetXEventSession("Profiler", liveConnection.ConnectionInfo);
-            Assert.NotNull(xeSession);
-            Assert.NotNull(xeSession.GetTargetXml());
-        }
+        }
+
+
+        private async Task<string> StartStandardSession(ProfilerService profilerService, string ownerUri)
+        {
+            const string sessionName = "ADS_Standard_Test";
+            var template = Newtonsoft.Json.JsonConvert.DeserializeObject<ProfilerSessionTemplate>(standardSessionJson);
+
+            var createParams = new CreateXEventSessionParams() { OwnerUri = ownerUri, SessionName = sessionName, Template = template };
+            var requestContext = new Mock<RequestContext<CreateXEventSessionResult>>();
+            requestContext.Setup(c => c.SendResult(It.IsAny<CreateXEventSessionResult>()))
+              .Returns<CreateXEventSessionResult>((result) => { return Task.FromResult(0); });
+            await profilerService.HandleCreateXEventSessionRequest(createParams, requestContext.Object);
+            return sessionName;
+        }
+
+
+
+#pragma warning disable JSON002 // Probable JSON string detected

```

Comment:
> JSON002 // Prob [](http://example.com/codeflow?start=23&length=16)

why does the compiler warn about a json string literal?

Reply:
I am not sure why the compiler is warning against the use of json string. But it looked harmless, hence suppressed the warning.

Reply:
We should always understand why we disable compiler warnings - just blindly disabling stuff in files doesn't help anyone. Either we should fix them or disable it in the editorconfig if it isn't useful. And when disabling please always include a reason why so we can have that context later on.  

In this case as far as I've been able to tell, this is warning that you don't have a comment like this telling the compiler that it's a JSON string

`/*lang=json,strict*/`

e.g. https://github.com/Azure/azure-functions-sql-extension/blob/main/test/Integration/SqlInputBindingIntegrationTests.cs#L128

Which lets it verify that the JSON is valid in the editor/at compile time. It's useful to have so I'd suggest adding it instead of disabling. 

(annoyingly I can't find any actual documentation on this whole lang comment stuff. The best I could find with a brief search was https://github.com/dotnet/docs/issues/30361...)

Reply:
I had the same problem, I couldn't find anything related to JSON002 error. But point noted.
Updated the code appropriately.